core/tests/performance_tests/test_config.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Performance test config."""
+
+TEST_DATA = [{
```

Comment:
hmm, I didn't understand you either.
I wasn't suggesting inline it to methods, I just suggested move TEST_DATA near to the place where it is used. By doing so, it will be easier to understand what AllTests is actually testing by looking to TEST_DATA.